lib/fuse_lowlevel.c

```diff
@@ -4051,6 +4051,13 @@ fuse_session_new_versioned(struct fuse_args *args,
 	int err;
 	struct fuse_session *se;
 	struct mount_opts *mo;
+	struct fuse_lowlevel_ops fallback_ops = { 0 };
+
+	if(op == NULL) {
+		fuse_log(FUSE_LOG_ERR, "fuse: warning: empty op list, falling back to default\n");
+		op = &fallback_ops;
+		op_size = sizeof(fallback_ops);
+	}
```

Comment:
To be honest, I don't see what the purpose is - why not return NULL? What is the purpose of a "null_op", if it won't do anything?